### ISSUE TITLE
AspectCodeFixProvider and MixinCodeProvider are throwing FileNotFoundException in VS 2022

### ISSUE THREAD

**Issue by jeremyj01** [pamidur/aspect-injector#179]
**Environment (please complete the following information):**
 - OS: Windows 10
 - Framework: net6.0
 - Type of application:  webapp
 - Version of AspectInjector: 2.7.2
 - IDE: Visual Studio 2022 (64-bit), Version 17.0.5

** Describe the bug **
The `AspectCodeFixProvider` and the `MixinCodeFixProvider` are throwing a `System.IO.FileNotFoundException`.

**To Reproduce**
Open a project using Visual Studio 2022 that contains one or more classes that use an aspect that was created using AspectInjector.  The advice for the overridden `Handle` method should be `Kind.Around`.  You will see the following errors at the top of VS studio:

'AspectCodeFixProvider' encountered an error and has been disabled. Show Stack Trace Enable Enable and ignore future errors 
'MixinCodeFixProvider' encountered an error and has been disabled. Show Stack Trace Enable Enable and ignore future errors 

This doesn't seem to affect the success of the build or runtime behavior, but it is concerning.

<details>
<summary>Exception stacktrace </summary>

```
System.IO.FileNotFoundException : Could not load file or assembly 'AspectInjector.Rules, Version=2.7.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
   at AspectInjector.Analyzer.CodeFixes.AspectCodeFixProvider.get_FixableDiagnosticIds()
   at Microsoft.CodeAnalysis.CodeFixes.CodeFixService.GetAndTestFixableDiagnosticIds(CodeFixProvider codeFixProvider)
   at System.Collections.Immutable.ImmutableInterlocked.GetOrAdd[TKey,TValue](ImmutableDictionary`2& location,TKey key,Func`2 valueFactory)
   at Microsoft.CodeAnalysis.Extensions.IExtensionManagerExtensions.PerformFunction[T](IExtensionManager extensionManager,Object extension,Func`1 function,T defaultValue)
```

</details>

**Comment by pamidur** [comment]
Hi @jeremyj01 , thanks for report, will look at shortly!

**Comment by jeremyj01** [comment]
Just a side note...this is one of my FAVORITE .net packages!

**Comment by pamidur** [comment]
@jeremyj01, thank you! It means much!

**Comment by pamidur** [comment]
Hi @jeremyj01 , I tried to reproduce this bug without success. Create aspect with Around advice and two classes using it within the same file. No errors.
I tend to believe (deducting from stacktrace you sent) that it was a Visual Studio glitch. In stackstace it mentions AspectInjector.Rules v2.7.0 though you're using 2.7.2 . Restarting VS should help.

If it doesn't, please send me a minimal file that reproduces the bug, that would help a lot!

**Comment by jeremyj01** [comment]
I back checked myself and realized I never merged my updates to using 2.7.2.  I'll take care of that shortly and report back.  Sorry if I raised an unnecessary alarm.

**Comment by jeremyj01** [comment]
@pamidur so...chock this one up to a PEBKAC error (**P**roblem **E**xists **B**etween **K**eyboard **A**nd **C**hair).  I updated a feature branch to 2.7.2, but never PR'd them to promote the updates to develop and finally master.  Once I did that, I no longer see the error.

Thanks for prodding me to recheck my work, my good dude! 😄
